thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/lib/FunctionReplayRunnable.java

```diff
@@ -64,7 +74,6 @@ public FunctionReplayRunnable(DetectionJobScheduler detectionJobScheduler, Anoma
     setForceBackfill(isForceBackfill);
     setTuningParameter(tuningParameter);
     setFunctionAutotuneConfigId(functionAutotuneConfigId);
-    setSpeedUp(true);
```

Comment:
The default speedup value is set in the base constructor. This value here is a duplicate.